tests/src/test/scala/com/evolutiongaming/cassandra/sync/CassandraSyncSpec.scala

```diff
@@ -19,25 +21,29 @@ import org.scalatest.wordspec.AnyWordSpec
 
 class CassandraSyncSpec extends AnyWordSpec with BeforeAndAfterAll with Matchers {
   import CassandraSyncSpec._
+  private lazy val cassandraContainer = CassandraContainer(
+    dockerImageNameOverride = DockerImageName.parse("cassandra:3.11.7"),
+  )
 
-  implicit val ioRuntime = implicits.global
+ // due to test structure we need to start the container before the test suite
+  cassandraContainer.start()
+
+  implicit val ioRuntime: IORuntime = implicits.global
+   private lazy val config = CassandraConfig.Default.copy(
+      contactPoints = Nel(cassandraContainer.containerIpAddress),
+      port = cassandraContainer.mappedPort(9042),
```

Comment:
is it possible to use some different port number? the default one conflicts with locally running Cassandra - one has to stop local version to run unit-tests

Reply:
it doesn't conflict. testcontainer assigns a random port and mappedPort returns the actual port that forwards to 9042.

Reply:
if I have some other docker-compose setup with Cassandra running and exposed on local port 9042. I will not be able to run this unit-test, right?

Reply:
Yes you would be able to run the test. As i mentioned, testcontainers assigns a random port and `mappedPort` returns the actual (random) port that forwards to 9042.

I just ran the tests locally while still having docker-compose running with Cassandra, with port 9042 occupied. It runs successfully.